gltfTutorial/gltfTutorial_005_MeshesTexturesImagesSamplers.md

```diff
@@ -0,0 +1,106 @@
+
+```javascript
+"nodes:" {
+    "exampleNode": {
+        "meshes" : [ "meshId0", "meshId1" ]
+        ...
+    }
+}
+```
+
+When the meshes are rendered, they are rendered with the [global transform](gltfTutorial_004_ScenesNodesCamerasAnimations.md#global-transforms-of-nodes) of the node that they are attached to. The same mesh may be "instanced" multiple times in a scene, by referencing it from different nodes that have different transforms.
+
+### Mesh primitives
+
+The mesh itself usually does not have any properties, but only contains an array of [`meshPrimitive`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-mesh.primitive) objects. These `meshPrimitive` objects are smaller parts or building blocks of a larger object:  
+
+```javascript
+"meshes": {
+    "meshId0": {
+        "primitives": [
+          {
+            "attributes": {
+              "POSITION": "accessorForPositions",
+              "NORMAL": "accessorForNormals",
+              "TEXCOORD_0": "accessorForTexCoords"
+            },
+            "indices": "accessorForIndices",
+            "mode": 4,
+            "material": "exampleMaterial"
+          }
+        ]
+    },
+    ...
+}
+```
+
+The `meshPrimitive` summarizes all information that describes how the respective part of the object will be rendered: It contains references to the geometry data in form of [`accessor`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-accessor) objects, as well as a reference to the [`material`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-material) that will be used for rendering. The details of the `accessor`s will be explained in the section about [Buffers, BufferViews and Accessors](gltfTutorial_007_BuffersBufferViewsAccessors.md). Here, only a simple, high-level description of accessors will be used, to give a basic understanding of how the geometry data is represented in the `meshPrimitive` objects of a glTF asset.
+
+#### Mesh primitive attributes
+
+In order to render the geometry of a `meshPrimitive`, the information about *what* should be rendered and *how* it should be rendered has to be combined. The information about *what* should be rendered is given by the actual geometry information. This is at least a set of vertices and certain attributes that belong to the vertices. The `attributes` dictionary of the `meshPrimitive` contains references to `acessor` objects. An `accessor` is an abstract source of data, and in the simplest case it can be imagined as a block of memory, together with some information about the data type and layout of this memory block. In the example above, the `attributes` contain references to three `accessors`. These accessors describe the positions and normal vectors for the vertices of the mesh, and the texture coordinates at these vertices:

```

Comment:
This explanation could potentially be clearer if, previously buffers, bufferViews, and accessors were introduced (in that order).